OpenNefia.Content/UI/UiHelpers.cs

```diff
@@ -0,0 +1,96 @@
+﻿using OpenNefia.Core.Locale;
+using OpenNefia.Core.Maths;
+using OpenNefia.Core.Prototypes;
+using OpenNefia.Core.Rendering;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenNefia.Content.UI
+{
+    public static class UiHelpers
+    {
+        public class DrawEntry : IDisposable
```

Comment:
Should rename this since it was moved out of its parent class. Maybe `UiBarDrawableState`.

Reply:
changed in 9eb449f